CHANGELOG.md

```diff
@@ -0,0 +1,50 @@
+# Changelog
+All notable changes to this project will be documented in this file.
+
+The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
+This project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html)
+and follows a [Backwards Compatability Policy]
+(https://docs.solana.com/developing/backwards-compatibility)
+
+## Unreleased
+### [1.17.0]
+* Changes
+  * Added a changelog!
+* Upgrade Notes
+
+## Released
+### 2023-05-31 - [1.16.0](https://github.com/solana-labs/solana/releases/tag/v1.16.0)
+
+## Adding to this Changelog
+### Audience
+* Entries in this log are intended to be easily understood by core contributors,
+consensus validator operators, rpc operators, and dapp developers.
+
+### Noteworthy
+* A change is noteworthy if it:
+  * Adds a feature gate, or
+  * Implements a SIMD, or
+  * Modifies API or ABI, or
```

Comment:
```suggestion
  * Modifies a public API, or
```
I removed ABI since that seems to be already covered by "Implements a SIMD", but I don't feel too strong